test/hotspot/jtreg/gc/g1/TestStringDeduplicationTools.java

```diff
@@ -383,6 +383,22 @@ public static void testTableRehash() throws Exception {
         output.shouldHaveExitValue(0);
     }
 
+    public static void testTableRehashFullGC() throws Exception {
+        // Test with StringDeduplicationRehashALot using full GCs
+        // Table resizing prevents table rehashing from happening, thus we use
+        // SmallNumberOfStrings to limit the number of table resizes.
+        OutputAnalyzer output = DeduplicationTest.run(SmallNumberOfStrings,
+                                                      DefaultAgeThreshold,
+                                                      FullGC,
+                                                      "-Xlog:gc,gc+stringdedup=trace",
+                                                      "-XX:+StringDeduplicationRehashALot");
+        output.shouldContain("Concurrent String Deduplication");
```

Comment:
Is there a reason you don't check `output.shouldContain("Full GC")`?

Reply:
Thanks, added this check. I copied it from `testTableRehash()` initially.